egs2/tedlium3/asr1/local/data.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $@"
+}
+SECONDS=0
+
+stage=1
+stop_stage=10
+
+log "$0 $*"
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    log "Error: no positional arguments are required."
+    exit 2
+fi
+
+. ./path.sh
+. ./cmd.sh
+
+data_type=legacy
+
+train_set="train"
+train_dev="dev"
+test_set="test"
+
+if [ ${stage} -le 1 ] && [ ${stop_stage} -ge 1 ]; then
+    log "stage 1: Data Download"
+    local/download_data.sh
+fi
+
+if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
+    log "stage 2: Data preparation"
+    local/prepare_data.sh ${data_type}
+    for dset in ${train_dev} ${test_set} ${train_set}; do
+        utils/data/modify_speaker_info.sh --seconds-per-spk-max 180 data/${dset}.orig data/${dset}
```

Comment:
utils/data/modify_speaker_info.sh can't be used because it depends on Kaldi's command.
We can just remove this block because we don't need to modify speaker info for end-to-end training.